recipe/meta.yaml

```diff
@@ -37,7 +36,7 @@ requirements:
 
 test:
   requires:
-    - scikit-learn >=0.18.0
+    - scikit-learn >=0.19.0
```

Comment:
Why was this bumped?

Reply:
Hm, good point. I must've misread the changelog for 0.7 (the compatibility layer for version *0.18* of scikit-learn). It could remain the same then.

Reply:
Mmmmk. The latest version should get installed anyways due to the `>=`.